stackit/internal/services/resourcemanager/project/resource.go

```diff
@@ -104,7 +104,7 @@ func (r *projectResource) Configure(ctx context.Context, req resource.ConfigureR
 // Schema defines the schema for the resource.
 func (r *projectResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
 	descriptions := map[string]string{
-		"main":                "Resource Manager project resource schema.",
+		"main":                "Resource Manager project resource schema. Must have a `region` specified in the provider configuration.",
```

Comment:
Isn't this one global?

Reply:
Yes it is, the `Configure` routine was reading `region` when it wasn't used. Removed